src/test/java/games/strategy/triplea/delegate/PlaceDelegateTest.java

```diff
@@ -28,10 +28,8 @@
     return gameData.getUnitTypeList().getUnitType(Constants.UNIT_TYPE_INFANTRY).create(count, player);
   }
 
-  @Override
-  @Before
-  public void setUp() throws Exception {
-    super.setUp();
+  @BeforeEach
+  public void setupPlaceDelegate() {
```

Comment:
Does JUnit 5 make guarantees on the order `@BeforeEach` methods will be called?  That is, does it ensure that a `@BeforeEach` method in the superclass will be called before a `@BeforeEach` method in a subclass?

**EDIT:** Nevermind, answered my own question:

> `@BeforeEach` methods are inherited from superclasses as long as they are not overridden. Furthermore, `@BeforeEach` methods from superclasses will be executed before `@BeforeEach` methods in subclasses.

Reply:
Yes, I should have provided a link to the JUnit docs